packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.tsx

```diff
@@ -16,6 +16,10 @@ export interface IGroupHeaderState {
 }
 
 export class GroupHeader extends BaseComponent<IGroupDividerProps, IGroupHeaderState> {
+  public static defaultProps: IGroupDividerProps = {
+    expandCollapseButtonProps: { 'aria-label': 'expand collapse group' }
```

Comment:
We don't want default values since we don't support localization in this repo. We'll need to take it as a prop to the component.